docs/tools/developer-tools/lab/README.mdx

```diff
@@ -7,6 +7,28 @@ sidebar_position: 80
 
 # Lab
 
-### [Lab](https://lab.stellar.org/?$=network$id=testnet&label=Testnet&horizonUrl=https:////horizon-testnet.stellar.org&rpcUrl=https:////soroban-testnet.stellar.org&passphrase=Test%20SDF%20Network%20/;%20September%202015;;)
+### [Stellar Lab](https://lab.stellar.org/?$=network$id=testnet&label=Testnet&horizonUrl=https:////horizon-testnet.stellar.org&rpcUrl=https:////soroban-testnet.stellar.org&passphrase=Test%20SDF%20Network%20/;%20September%202015;;)
```

Comment:
We can omit the network config params in the URL to make it easier to read. Tesnet will be selected by default.

```suggestion
### [Stellar Lab](https://lab.stellar.org/)
```